x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/__mocks__/overview_logic.mock.ts

```diff
@@ -29,14 +29,14 @@ export const mockLogicValues = {
   flashMessages: {},
 } as IOverviewValues;
 
+export const mockLogicActions = {
+  initializeOverview: jest.fn(() => ({})),
+};
+
 jest.mock('kea', () => ({
   ...(jest.requireActual('kea') as object),
-  useActions: jest.fn(() => ({
-    initializeOverview: jest.fn(() => ({})),
```

Comment:
The reason why this was returning 0 calls is Javascript obj reference shenanigans. In this current code, `useActions()` returns a **brand new obj reference** containing a new `initializeOverview: jest.fn()` each time. Because we were [destructuring a new `useActions()` on line 42](https://github.com/elastic/kibana/pull/71756/commits/0d4e519df5165f8d753843e49f754adf4a5799e6#diff-075f4ea9ecb8171d79b679b9f41f8f2eL42), it wasn't referencing the _same_ `initializeOverview` that was called on mount() and as such didn't have the correct # of calls.

The fix is to have `useActions()` to return a reference to a single static `initializeOverview: jest.fn()` so that the number of calls is correctly tracked instead of being overridden. We do this by declaring a static [mockLogicActions obj](https://github.com/elastic/kibana/pull/71756/commits/0d4e519df5165f8d753843e49f754adf4a5799e6#diff-73f59d36576c863cfc1c82bfa1fe7268R32-R35) that is easily reference-able (in theory I could have also just exported a single `intializeOverview` var and that would also have worked just fine, but I figured this followed your current patterns better).